cylc/flow/task_pool.py

```diff
@@ -1908,26 +1908,33 @@ def set_prereqs_and_outputs(
             flow_descr: description of new flow
 
         """
-        flow_nums = self._get_flow_nums(flow, flow_descr)
-        if flow_nums is None:
-            # Illegal flow command opts
-            return
-
```

Comment:
(Maybe this was the "redundant option checking" that I removed, although it seems a bit trivial.)